code/modules/projectiles/projectile.dm

```diff
@@ -199,8 +199,8 @@
 			M.Turn(Angle)
 			transform = M
 
-			var/Pixel_x=round(sin(Angle)+16*sin(Angle)*2)
-			var/Pixel_y=round(cos(Angle)+16*cos(Angle)*2)
+			var/Pixel_x=round((sin(Angle)+16*sin(Angle)*2), 1)	//Without the 1 it rounds to -1 for cos because byond's functions are shit.

```

Comment:
this... sounds like a bug that should be reported.
Check the ref for ```cos()``` and ```sin()``` to make sure it's not intended behaviour.